Makefile.defs

```diff
@@ -512,11 +508,6 @@ endif
 ifdef DARWIN_AARCH64_HOST
 # At the moment of writing, apple-m1 is the lowest common denominator for Apple silicon chips. Should work for M2 as well
 	HOST_ARCH:=-mcpu=apple-m1
-	BREW_PREFIX:=$(shell command brew --prefix 2>/dev/null)
-	ifneq (,$(BREW_PREFIX))
-		export DARWIN_AARCH64_HOST_CFLAG:=-I$(BREW_PREFIX)/include
-		export DARWIN_AARCH64_HOST_LDFLAG:=-L$(BREW_PREFIX)/lib
-	endif
```

Comment:
Is this unused in the sense that it's not necessary on M1/M2 and/or newer Brew? That is, this isn't comparable to some internal `CLANG=1` remnant.

Reply:
This was added in 30bf21c674, but not necessary anymore after d1910575bf240a14d53b5649e9301fec21f2d950 (and I should have removed it there).